backend/migrations-backdev/20220125143932-backdev_col.js

```diff
@@ -1446,18 +1447,16 @@ module.exports = {
       },
     );
     await db.createCollection('clientCourses');
-    await db.createCollection('accessTokenBlacklist');
```

Comment:
Why removing this collection?

Reply:
idk, i took some data from @zhuk-vezunchikz  migration